index.html

```diff
@@ -215,6 +206,7 @@ <h2 class="title2">About me</h2>
                   </p>
                   <button type="button" id="resume">Get My Resume</button>
                 </div>
+                <div class="line2"></div>
                 <div id="skills">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_10](https://user-images.githubusercontent.com/69638013/161400409-41cc3acd-ec64-4c35-83f4-b08a6260ec77.png)  | ![Screenshot_11](https://user-images.githubusercontent.com/69638013/161400416-df77cbb2-fc50-4682-a97a-5ec3a7846185.png)

- [ ] Kindly add spacing between the icons and the header tags, this will make your design look like the figma design 👍🏿

Reply:
- [ ] As suggested by the previous reviewer, Kindly make sure to add spacing between the icons and the header tags, this will make your design look like the Figma design

Reply:
- [ ] Please as required by the previous reviewer, you can consider adding space between the `icons` and the header tags

- See the Figma design:

![good1](https://user-images.githubusercontent.com/79058364/161404586-1bfb0d3b-bd74-42ff-a9d3-bce3a6bcfb2a.png)